command/validator/stake/params.go

```diff
@@ -30,6 +36,16 @@ func (sp *stakeParams) validateFlags() (err error) {
 		return fmt.Errorf("failed to parse json rpc address. Error: %w", err)
 	}
 
+	if err := types.IsValidAddress(sp.stakeToken); err != nil {
+		return fmt.Errorf("stake token address is not a valid address: %w", err)
+	}
+
+	sp.stakeTokenAddr = types.StringToAddress(sp.stakeToken)
+
+	if sp.stakeTokenAddr == types.ZeroAddress {
+		return errStakeTokenIsZeroAddress
+	}
```

Comment:
We have this code on the multiple places. Probably it is worth considering to expand `IsValidAddress` to return `types.Address` in case address is valid and also to check for zero address (however we should specify if `0x0` address is valid or not via parameter). However we can do that through a separate PR.

Reply:
Ok, we will fix it with separate PR.